src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[],
+  tempTodo: Todo | null,
+  loadingTodosId: number[],
+  onTodoDelete: (todoId: number) => void;
+  onTodoUpdate: (todoId: number, updatedTodo: Partial<Todo>) => void;
+};
+
+export const TodoList: FC<Props> = ({
+  todos,
+  tempTodo,
+  loadingTodosId,
+  onTodoDelete,
+  onTodoUpdate,
+}) => {
+  return (
+    <section className="todoapp__main">
+      <TransitionGroup>
+        {todos.map(todo => (
+          <CSSTransition
+            key={todo.id}
+            timeout={300}
+            classNames="item"
+          >
+            <TodoInfo
+              key={todo.id}
```

Comment:
Remove the duplicate key prop on TodoInfo component, as it is already provided on CSSTransition component on line 29.